### PR TITLE
BZ2174226: CNV 4.11 RN known issue

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Version(s): 4.11

Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2174226

Link to docs preview: https://56946--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-11-release-notes.html#virt-4-11-known-issues

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sjhala-ccs** [comment]
@kbidarkar or @dshchedr Can you please review this release note PR as well? Thanks.

**Review by lahinson — COMMENTED** [review]
Looks good overall! I had just a few minor suggestions and comments.

File: virt/virt-4-11-release-notes.adoc [hunk 1]
```diff
@@ -199,6 +199,9 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 
 * In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV Reenlightenment enabled cannot be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
 
+* When using two pods with different SELinux contexts, VMs with the `ocs-storagecluster-cephfs` storage class fail to migrate and the VM status changes to `Paused`. This is because both pods try to access the shared `ReadWriteMany` CephFS volume at the same time. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2092271[*BZ#2092271*])
+** As a workaround, use the `ocs-storagecluster-ceph-rbd` storage class for live migrating VMs on a cluster with Ceph storage.
```
[the remark this answers is not in the document]
Comment (sjhala-ccs) [reply virt/virt-4-11-release-notes.adoc:203]: @lahinson Thank you for your review comments. Regarding your suggestion to hyphenate, we use the terms `live migration` and `live migrate` throughout the OpenShift Virtualization docs, so I am going to keep it as is.

**Pull request merged**